packages/api/src/external/fhir-to-cda-converter/connector.ts

```diff
@@ -1,8 +1,12 @@
 import { Bundle } from "../../routes/medical/schemas/fhir";
 
+/**
+ * @typeParam splitCompositions - boolean to indicate whether to split the bundle based on the Compositions
+ * prior to FHIR-to-CDA conversion. Should be set to `false` for CCD generation.
+ */
 export type FhirToCdaConverterRequest = {
   cxId: string;
-  toSplit: boolean;
+  splitCompositions: boolean;
```

Comment:
With `@typeParam` on the main type:

<img width="378" alt="image" src="https://github.com/metriport/metriport/assets/2132564/8e1e98d7-2965-4950-a4d1-1d498bf698f1">

With a TSDoc comment on the property itself:

<img width="879" alt="image" src="https://github.com/metriport/metriport/assets/2132564/300a3246-84e1-4b0c-860f-c626bfd00824">

Something like this:

<img width="649" alt="image" src="https://github.com/metriport/metriport/assets/2132564/8dfa9ac5-fcdc-4cdf-b5ca-2b60223b6aac">

Reply:
Oh cool! I didn't know that each param could have its own TSDoc like that! TIL!